### PR TITLE
docs: add cheermotes and codes markdown docs

### PR DESCRIPTION
**Pull request by tadite**
# Prerequisites for Code Changes
* [x] This pull request follows the code style of the project
* [x] I have tested this feature

### Issues Fixed 
* [Issue #203]

### Additional Information 
Added some markdown docs for cheermotes and codes methods

### CODE REVIEW

**Review by iProdigy — CHANGES_REQUESTED** [review]
Thank you very much for your PR! If you have the time, could you apply some of these quick changes?

File: docs/content/rest-helix/_index.md [hunk 1]
```diff
@@ -90,3 +91,7 @@ Videos:
 
 - [Videos -> Search](./videos-get)
 - Get Webhook Subscriptions
+
+Entitlements:
+- [Entitlements -> Get Code Status](./entitlements-codes.md)
+- [Entitlements -> Redeem Code](./entitlements-redeem-codes.md)
```
[the remark this answers is not in the document]
Comment (tadite) [reply docs/content/rest-helix/_index.md]: Fixed .md and moved bullet points to existed section

File: docs/content/rest-helix/bits-cheermotes.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @Param("token") String authToken,
+    @Param("broadcaster_id") String broadcasterId
+);
+```
+
+*Required Parameters*
+
+| Name          | Type      | Description  |
+| ------------- |:---------:| -----------------:|
+| authToken     | string    | User Auth Token |
```
[the remark this answers is not in the document]
Comment (tadite) [reply docs/content/rest-helix/bits-cheermotes.md]: Fixed

File: docs/content/rest-helix/entitlements-redeem-codes.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    @Param("code") List<String> code,
+    @Param("user_id") Integer userId
+);
+```
+
+*Required Parameters*
+
+| Name          | Type      | Description  |
+| ------------- |:---------:| -----------------:|
+| authToken     | string    | User Auth Token |
```
[the remark this answers is not in the document]
Comment (tadite) [reply docs/content/rest-helix/entitlements-redeem-codes.md]: Changed

**Review by iProdigy — APPROVED** [review]
Thank you for your contribution to the docs!

**Pull request merged**